### ISSUE TITLE
Storage mismatch @testnet's block 1182710

### ISSUE THREAD

**Issue by roman-khimov** [nspcc-dev/neo-go#972]
```
block 1182710: value mismatch for key c2700ebed4ba0fabd7500f3c705251fa7d01dbd75050e271306f6cd9841735ba43d0a80200f7ef537c38a20500000000000000000009: 002d0100000001000000041c0000000000000f000000027ffffffe0000001a7fffffff0000001a7ffffffa0000000000 vs 002901000000010000000418000000000000027ffffffe0000001a7fffffff0000001a7ffffffa0000000000
```
Ours:
```
{
   "jsonrpc" : "2.0",
   "id" : 1,
   "result" : {
      "executions" : [
         {
            "stack" : [
               {
                  "type" : "Boolean",
                  "value" : true
               }
            ],
            "trigger" : "Application",
            "notifications" : [
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "type" : "ByteArray",
                           "value" : "00000000010000000418000000000000027ffffffe0000001a7fffffff0000001a7ffffffa00000000"
                        }
                     ]
                  }
               },
               {
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "type" : "ByteArray",
                           "value" : "436f6e7472616374"
                        }
                     ]
                  },
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2"
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "0",
                           "type" : "Integer"
                        }
                     ]
                  }
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "1",
                           "type" : "Integer"
                        }
                     ]
                  }
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "value" : [
                        {
                           "value" : "04",
                           "type" : "ByteArray"
                        }
                     ],
                     "type" : "Array"
                  }
               },
               {
                  "state" : {
                     "value" : [
                        {
                           "value" : "24",
                           "type" : "Integer"
                        }
                     ],
                     "type" : "Array"
                  },
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2"
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "000000027ffffffe0000001a7fffffff0000001a7ffffffa",
                           "type" : "ByteArray"
                        }
                     ]
                  }
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "value" : [
                        {
                           "type" : "Integer",
                           "value" : "0"
                        }
                     ],
                     "type" : "Array"
                  }
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "000000027ffffffe0000001a7fffffff0000001a7ffffffa",
                           "type" : "ByteArray"
                        }
                     ]
                  }
               },
               {
                  "state" : {
                     "value" : [
                        {
                           "value" : "57697468206465706f7369743f",
                           "type" : "ByteArray"
                        }
                     ],
                     "type" : "Array"
                  },
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2"
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "value" : [
                        {
                           "type" : "Boolean",
                           "value" : false
                        }
                     ],
                     "type" : "Array"
                  }
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "53554343455353",
                           "type" : "ByteArray"
                        }
                     ]
                  }
               }
            ],
            "vmstate" : "HALT",
            "contract" : "0x5c81a4488a9acf544e6e1234d6608f0975f6f330",
            "gas_consumed" : "7.635"
         }
      ],
      "txid" : "0x121d272b4e01795fe1c165297849d19f174ef731e9722600c89c90aa7e619456"
   }
}
```
Theirs:
```
{
   "result" : {
      "executions" : [
         {
            "vmstate" : "HALT",
            "notifications" : [
               {
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "00000000010000000418000000000000027ffffffe0000001a7fffffff0000001a7ffffffa00000000",
                           "type" : "ByteArray"
                        }
                     ]
                  },
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2"
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "value" : [
                        {
                           "type" : "ByteArray",
                           "value" : "436f6e7472616374"
                        }
                     ],
                     "type" : "Array"
                  }
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "0",
                           "type" : "Integer"
                        }
                     ]
                  }
               },
               {
                  "state" : {
                     "value" : [
                        {
                           "value" : "1",
                           "type" : "Integer"
                        }
                     ],
                     "type" : "Array"
                  },
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2"
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "type" : "ByteArray",
                           "value" : "04"
                        }
                     ]
                  }
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "value" : [
                        {
                           "type" : "Integer",
                           "value" : "24"
                        }
                     ],
                     "type" : "Array"
                  }
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "value" : [
                        {
                           "type" : "ByteArray",
                           "value" : "000000027ffffffe0000001a7fffffff0000001a7ffffffa"
                        }
                     ],
                     "type" : "Array"
                  }
               },
               {
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "type" : "Integer",
                           "value" : "0"
                        }
                     ]
                  },
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2"
               },
               {
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "000000027ffffffe0000001a7fffffff0000001a7ffffffa",
                           "type" : "ByteArray"
                        }
                     ]
                  },
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2"
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "57697468206465706f7369743f",
                           "type" : "ByteArray"
                        }
                     ]
                  }
               },
               {
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2",
                  "state" : {
                     "value" : [
                        {
                           "value" : false,
                           "type" : "Boolean"
                        }
                     ],
                     "type" : "Array"
                  }
               },
               {
                  "state" : {
                     "type" : "Array",
                     "value" : [
                        {
                           "value" : "53554343455353",
                           "type" : "ByteArray"
                        }
                     ]
                  },
                  "contract" : "0xd7db017dfa5152703c0f50d7ab0fbad4be0e70c2"
               }
            ],
            "contract" : "0x5c81a4488a9acf544e6e1234d6608f0975f6f330",
            "gas_consumed" : "7.635",
            "trigger" : "Application",
            "stack" : [
               {
                  "type" : "Boolean",
                  "value" : true
               }
            ]
         }
      ],
      "txid" : "0x121d272b4e01795fe1c165297849d19f174ef731e9722600c89c90aa7e619456"
   },
   "jsonrpc" : "2.0",
   "id" : 1
}
```

**Comment by roman-khimov** [comment]
Contract: https://github.com/elightWorkspace/SmartContract

**Comment by roman-khimov** [comment]
Fixed by #975.
